website/src/main/java/com/google/step/data/OrganizationInfo.java

```diff
@@ -0,0 +1,61 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import java.util.Objects;
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import org.apache.commons.validator.UrlValidator;
+
+public final class OrganizationInfo {
+  private final Entity entity;
```

Comment:
I am against making global variables, the Entity class's scope could be reduced

Reply:
Could you elaborate on the benefit of that? As of now I like having it as a data member to keep everything locked inside the class instance as opposed to the extra clutter of parameter passing in doPost()

Reply:
nvm what you are doing here makes sense, ignore what I said...